modules/profile-store/pages/index.adoc

```diff
@@ -0,0 +1,177 @@
+= Building the right indexes for querying a User Profile Store
```

Comment:
I've been missing some context here. We usually reserve **index.adoc** for introduction/set-up instruction files, for this tutorial it's already the install page https://docs.couchbase.com/tutorials/profile-store/install.html

Suggest renaming this file to **building-indexes.adoc** or something similar to avoid the possible confusion.